web_src/js/components/RepoActivityTopAuthors.vue

```diff
@@ -45,8 +45,8 @@ const styleElement = useTemplateRef('styleElement');
 const altStyleElement = useTemplateRef('altStyleElement');
 
 onMounted(() => {
-  const refStyle = window.getComputedStyle(styleElement.value);
-  const refAltStyle = window.getComputedStyle(altStyleElement.value);
+  const refStyle = window.getComputedStyle(styleElement.value!);
+  const refAltStyle = window.getComputedStyle(altStyleElement.value!);
```

Comment:
I don't think it's right to keep adding `!` everywhere.

If you are refactoring, then it should follow the best practice and make the code readable / maintainable -- unless you will do quick following up changes to fix them in short time.

Just like the "jQuery removal", adding a lot `?` only makes code difficult to read or maintain. At last I have to clean up them one by one to really finish the refactoring.


-----

For this one, maybe the proper fix is to use `useTemplateRef<T>`

And a lot of other places also need a proper fix.